agir/front/components/app/Footer/Footer.js

```diff
@@ -124,31 +120,31 @@ export const Footer = (props) => {
           <div>
             <h3>Action populaire</h3>
             <p>
-              <Link route="donationLanding">Faire un don</Link>
-              <Link route="eventMap">Carte des événements</Link>
-              <Link route="groupMap">Carte des groupes</Link>
+              <Link route="donationLanding">{_("Faire un don")}</Link>
+              <Link route="eventMap">{_("Carte des événements")}</Link>
+              <Link route="groupMap">{_("Carte des groupes")}</Link>
               <Link route="materiel" target="_blank">
-                Commander du matériel
+               {_(" Commander du matériel")}
               </Link>
             </p>
           </div>
 
           <div>
-            <h3>Liens utiles</h3>
+            <h3>{_("Liens utiles")}</h3>
             <p>
               {isSignedIn ? (
-                <Link route="logout">Se déconnecter</Link>
+                <Link route="logout">{_("Se déconnecter")}</Link>
               ) : (
-                <Link route="login">Se connecter</Link>
+                <Link route="login">{_("Se connecter")}</Link>
               )}
-              <Link route="help">Besoin d'aide&nbsp;?</Link>
-              <Link route="legal">Mentions légales</Link>
-              <Link route="contact">Contact</Link>
+              <Link route="help">{_("Besoin d'aide&nbsp;?")}</Link>
```

Comment:
HTML characters will not work anymore in the string literal. For the non-breaking space you can just replace it with a normal space, since it is going to be translated anyway :
```suggestion
              <Link route="help">{_("Besoin d'aide ?")}</Link>
```